packages/server/src/synchronizer/rules/config.ts

```diff
@@ -25,7 +25,20 @@ export async function copyConfig(entries: string[], srcPath: string, destPath: s
 
   await writeFile(resolve(destPath, 'blitz.config.js'), Buffer.from(fileContents))
 
-  const nextConfigShellTpl = `
+  // Zeit now adds configuration needed for Now, like serverless target,
+  // so we need to keep and use that
+  if (process.env.NOW_BUILDER) {
+    rename(resolve(srcPath, 'next.config.js'), resolve(destPath, 'next-zeit.config.js'))
```

Comment:
Problem is you are missing an await here I think

Reply:
Also we can't rename here. We can't alter the source folder as this is run after the entries have been taken.

Reply:
Will push a fix